tests/cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
should there be:
```
set -o errexit
set -o nounset
set -o pipefail
```
?

Reply:
The cleanup script should be more tolerant of failure. But we should bring back nounset, because that's clearly a script error.

Done.